doc/source/fluid_properties/create_fluids.rst

```diff
@@ -10,8 +10,9 @@ Fluid from Library
 ==================
 
 In the fluid library some default fluids are already implemented. Currently it is
-possible to work with the default fluids high or low caloric natural
-gas (hgas or lgas), water and air. One of these default fluids can be created and added to an
+possible to work with the default fluids high and low caloric natural
+gas (hgas and lgas), water, air, carbon dioxide, ethane, hydrogen, methane, nitrogen and oxygen.
```

Comment:
carbon dioxide, ethane, hydrogen, methane, nitrogen and oxygen are available in the properties folder, but not yet implemented in call_lib. So I suggest to not mention them in the documentation at this point.
```suggestion
gas (hgas and lgas), water and air.
```